app/controllers/admin/referents_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
     @user = policy_scope(User).find(index_params[:user_id])
     authorize(@user, :update?)
     @referents = policy_scope(@user.agents).distinct.order(:last_name)
-    @agents = policy_scope(Agent)
+    @agents = policy_scope(Agent).joins(:organisations).where("organisations.id": current_organisation.id)
```

Comment:
```suggestion
    @agents = policy_scope(Agent).joins(:organisations).where(organisations: current_organisation)
```
? ça produit le même SQL

Ou alors un coup plus loin:

```suggestion
    @agents = policy_scope(Agent).merge(current_organisation.agents)
```
qui à l’avantage de mieux exprimer ce qu’on veut (enfin je trouve) et qui fait une jointure plus simple